modelforge/potential/parameters.py

```diff
@@ -32,18 +32,19 @@ class PerAtomEnergy(ParametersBase):
 class ANI2xParameters(ParametersBase):
     class CoreParameter(ParametersBase):
         angle_sections: int
-        radial_max_distance: Union[str, unit.Quantity]
-        radial_min_distance: Union[str, unit.Quantity]
+        maximum_interaction_radius: Union[str, unit.Quantity]
+        minimum_interaction_radius: Union[str, unit.Quantity]
         number_of_radial_basis_functions: int
-        angular_max_distance: Union[str, unit.Quantity]
-        angular_min_distance: Union[str, unit.Quantity]
+        maximum_interaction_radius_for_angular_features: Union[str, unit.Quantity]
+        minimum_interaction_radius_for_angular_features: Union[str, unit.Quantity]
         angular_dist_divisions: int
+        activation_function: str
```

Comment:
I think it would be best to define the activation function as an Enum rather than str. 

`class ActivationFunction(str, Enum):
    ReLU = "ReLU"
    CeLU = "CeLU"
    Sigmoid = "Sigmoid"
    Softmax = "Softmax"
    ShiftedSoftplus = "ShiftedSoftplus"
    SiLU = "SiLU"
    Tanh = "Tanh"
    LeakyReLU = "LeakyReLU"
    ELU = "ELU"
    `